File: python/udemy/3-estrutura-sequencial/duracao.py
# Fazer um programa para ler uma duração de tempo em segundos, daí imprimir na tela esta duração no formato horas:minutos:segundos.

segs = int(input('Duração de tempo em segundos: '))

horas = segs // 3600
resto = segs % 3600

minutos = resto // 60
segundos = resto % 60

print(f'A duração {segs}s corresponde a {horas}h:{minutos}min:{segundos}s.')

File: c/udemy/3-sequencial/duracao.c
// Fazer um programa para ler uma duração de tempo em segundos, daí imprimir na tela esta duração no formato horas:minutos:segundos

#include <stdio.h>

int main() {

    int h, min, seg, duracao, resto;

    printf("Tempo em segundos: ");
    scanf("%d", &duracao);

    h = duracao / 3600;
    resto = duracao % 3600;

    min = resto / 60;
    seg = resto % 60;

    printf("%d:%d:%d", h, min, seg);

    return 0;

}